sympy/matrices/tests/test_commonmatrix.py

```diff
@@ -1268,3 +1271,52 @@ def test_jordan_form():
     P, J = m.jordan_form()
     assert all(isinstance(x, Float) or x == 0 for x in P)
     assert all(isinstance(x, Float) or x == 0 for x in J)
+
+def test_singular_values():
+    x = Symbol('x', real=True)
+
+    A = EigenOnlyMatrix([[0, 1*I], [2, 0]])
+    assert A.singular_values() == [2, 1]
+
+    A = eye(3)
+    A[1, 1] = x
+    A[2, 2] = 5
+    vals = A.singular_values()
+    assert 1 in vals and 5 in vals and abs(x) in vals
```

Comment:
This is what SymPy will give:
```
>>> vals
[Abs(x), 5, 1]
```
The other asserts take the form `assert  ,,, == ...`, so I was trying to find some reason why this should be different. The absence of `Abs` was the only one I could think of. It is no big deal but the single equality is probably more efficient than the three separate conditions. It that is so, then this PR might be a good place for a change.